src/main/java/com/microsoft/spring/data/gremlin/conversion/script/GremlinScriptLiteral.java

```diff
@@ -19,20 +19,23 @@
      */
     List<String> generateInsertScript(GremlinSource source);
 
-
     /**
      * Generate the deleteAll query from source (Vertex, Edge or Graph).
      */
     List<String> generateDeleteAllScript(GremlinSource source);
 
-
     /**
-     * Generate the findById query from source (Vertex, Edge or Graph).
+     * Generate the findById query from source (Vertex, Edge).
```

Comment:
Why this `or Graph` is removed in the comment, but some other functions below are kept?

Reply:
From the gremlin server side, we cannot find the graph by Id. As the domains in a graph will lose the type. We cannot retrieve that for now. The comments is not verify clear before, fix it here.